package.json

```diff
@@ -20,14 +20,23 @@
   },
   "activationEvents": [
     "onDebug",
-    "onCommand:cdt.gdb.memory.open"
+    "onCommand:cdt.gdb.memory.open",
+    "onCommand:cdt.debug.resumeAllSession"
   ],
   "contributes": {
     "commands": [
       {
         "category": "GDB",
         "title": "Open Memory Browser",
         "command": "cdt.gdb.memory.open"
+      },
+      {
+        "command": "cdt.debug.resumeAllSession",
+        "title": "Resume All Sessions",
+        "icon": {
+          "light": "images/ResumeAll_Light.png",
```

Comment:
The icons are much better, but they still look distinctly different from the rest of the buttons. Do you see this difference too on your machine? 

![image](https://user-images.githubusercontent.com/679236/217983521-9037ed14-59a2-4e30-8805-46fc46f27158.png)

This is a crop from your screenshot which seems to show the same as I am seeing, in particular that the triangle is a different shape:

![image](https://user-images.githubusercontent.com/679236/217983708-114443ed-a613-4193-bfd2-b46fd3c9b385.png)

Is this down to how VSCode renders the built-in icons vs the contributed ones? Is it possible to fix this?

Reply:
I have updated the new images.

Reply:
The new images look great and appear to seamlessly match the style of the existing icons.